src/Arbitrary.hs

```diff
@@ -492,7 +634,7 @@ instance Arbitrary Document where
     arbitrary = arbMarkdown
 
     shrink (Markdown "") = []
-    shrink _ = [Markdown ""]
+    shrink _ = [Markdown "", Markdown "x"]
```

Comment:
The three big test partitions are the empty, one, n characters in a document. If a test fails on a non-empty string, printing out one character long document is nicer.